web/frontend/src/components/camera/camera.svelte

```diff
@@ -16,6 +16,9 @@ let imgEl: HTMLImageElement;
 
 let cameraFrameIntervalId = -1;
 
+// operationActive provides simple overlap protection in the setInterval callback.
+let operationActive = false;
```

Comment:
This LGTM - nice idea!

Reply:
Gah, I just remembered we have a utility for this that we should use: [`setAsyncInterval`](https://github.com/viamrobotics/rdk/blob/2242ac71dc89ecd116f1a2fa5d74955a55f0a946/web/frontend/src/lib/schedule.ts). Do you mind updating to that?

Reply:
done in 09c94050...03dd9469, way cleaner with library function

(also sorry I missed this notification earlier)

things to note about this change:
- I added docs + an ignore error param to `setAsyncInterval`
- I had to fight the linter to pass a closure which returned an async function, lmk if there's a better way to do this